cmd/gtfTools/filter.go

```diff
@@ -71,8 +74,29 @@ func gtfFilter(s FilterSettings) {
 	}
 
 	records := gtf.Read(s.InFile)
-	for currGene := range records {
+	for currGene := range records { // for each gene
 		pass = true
+
+		if s.CodingTranscript { // option to detect coding transcript
+			for _, currTranscript := range records[currGene].Transcripts { // for each transcript
+				if len(currTranscript.Exons) == 0 { // if transcript has no exon, then transcript is non-coding
```

Comment:
Craig's comment 1: now the pass is for gene, but decision to pass is based on gene's last transcript. 
Christi's suggestion: Set pass to false when go into transcript. Only pass transcripts that are coding.
Craig's comment 2: Line 82 if may not be necessary.